docs/extending/creating_custom_dropdowns.md

```diff
@@ -5,83 +5,128 @@ Follow the steps below to learn how to integrate this component to fit it to you
 
 ## Prepare custom drop-down structure
 
-First prepare the component HTML code structure in the template inside the `content` section:
-
-```html hl_lines="2 11 12"
-<div class="ibexa-custom-dropdown">
-    <select class="ibexa-custom-dropdown__select" hidden multiple>
-        <option value="1">Option 1</option>
-        <option value="2">Option 2</option>
-        <option value="3">Option 3</option>
-        <option value="4">Option 4</option>
-    </select>
-    <div class="ibexa-custom-dropdown__wrapper">
-        <ul class="ibexa-custom-dropdown__selection-info"></ul>
-        <ul class="ibexa-custom-dropdown__items ibexa-custom-dropdown__items--hidden">
-            <li data-value="" class="ibexa-custom-dropdown__item" disabled>Select an option</li>
-            <li data-value="1" class="ibexa-custom-dropdown__item">Option 1</li>
-            <li data-value="2" class="ibexa-custom-dropdown__item">Option 2</li>
-            <li data-value="3" class="ibexa-custom-dropdown__item">Option 3</li>
-            <li data-value="4" class="ibexa-custom-dropdown__item">Option 4</li>
-        </ul>
-    </div>
-</div>
+First prepare the component structure and place it in the template inside the `content` section. See the example:
+
+```twig
+{% include '@ibexadesign/ui/component/dropdown.html.twig' with {
+    source,
+    choices,
+    preferred_choices,
+    value,
+    multiple,
+    translation_domain,
+    custom_form,
+    class,
+    placeholder,
+    custom_init,
+} %}
```

Comment:
Well... bad news, since we first talked about it, dropdown was expanded a little and there are new properties that can be passed. :)

`is_disabled` - allows to disable dropdown
`is_hidden` - allows to hide whole widget
`is_small` - adjust height of widget (from 48px to 32px)
`is_ghost` - changes layout of widget, it removes all borders and backgrounds (similar to buttons modifier)
`min_search_items` - default 5, minimum number of options that has to be passed to show search inside dropdown
`selected_item_label` - allows to set constant label for widget, by default visible label shows currently selected options

ping me on Slack if you need more info or sth is unclear